### PR TITLE
Fix phabricator example

### LINKED ISSUE #6406: Phabricator example does not set proper minion scopes on GCE
As a result cloudsql_authenticator reports the following error:

ERROR: (gcloud.sql.instances.patch) Insufficient Permission

### PR DESCRIPTION
**Pull request by fgrzadkowski**
This commit has several fixes/simplifications:
1. Deletes `cloudsq_authenticator` in favor of manual command to authorize all nodes from the cluster to talk to CloudSQL. Previous solution didn't work for a default cluster setup as it required different node scopes, which cannot be update on GCE after creation.
2. Updates how to get external IP for a service
3. Adds required env variables to phabricator RC

Fixes #6406

### CODE REVIEW

File: examples/phabricator/README.md [hunk 1]
```diff
@@ -43,16 +43,36 @@ This example assumes that you have a basic understanding of kubernetes [services
...
-$ hack/dev-build-and-up.sh
+$ cluster/kube-up.sh
 ```
 
 ### Step One: Set up Cloud SQL instance
 
 Follow the [official instructions](https://cloud.google.com/sql/docs/getting-started) to set up Cloud SQL instance.
 
-In the remaining part of this example we will assume that your instance is named "phabricator-db", has IP 173.194.242.66 and the password is "1234".
+In the remaining part of this example we will assume that your instance is named "phabricator-db", has IP 173.194.105.37, is listening on port 3306 and the password is "1234".
```
Comment (thockin) [inline examples/phabricator/README.md]: it would be easier to read this if you made up an IP that was easy to remember :)

Comment (fgrzadkowski) [inline examples/phabricator/README.md]: Good point :) I used real values to make sure this script actually works.

**Comment by fgrzadkowski** [comment]
@thockin Comment addressed. PTAL.

**Comment by brendandburns** [comment]
examples test is failing.

Run
```
./hack/test-go.sh examples
```

**Comment by fgrzadkowski** [comment]
@wojtek-t Can you please LGTM this. I pushed new version which removed LGTM. No changes in the example - I only fixed the test.

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**